resources/scripts/ttx_diff.py

```diff
@@ -138,6 +138,25 @@ def copy(old, new):
     return new
 
 
+def name_id_to_name(ttx, xpath, attr):
+    id_to_name = {
+        el.attrib["nameID"]: el.text.strip()
+        for el in ttx.xpath("//name/namerecord[@platformID='3' and @platEncID='1' and @langID='0x409']")
+    }
+    for el in ttx.xpath(xpath):
+        if attr not in el.attrib:
+            continue
+        print(f"{el.attrib[attr]} => {id_to_name[el.attrib[attr]]}")
+        el.attrib[attr] = id_to_name[el.attrib[attr]]
```

Comment:
I don't understand why are you modifying the tree here? you want a NamedInstance's the subfamilyNameID attribute to contain the text string itself instead of the nameID index? Doesn't fonttools ttx already add that very same name string (as a xml comment) next to every nameID that is referenced?

Reply:
For font-specific ids that are not assigned by the spec (e.g. >255) I don't want to care if the id differs. Concretely, I don't care about this:
![image](https://github.com/googlefonts/fontmake-rs/assets/6466432/9f5d715c-5921-4c5f-b78a-1660f75b1083)

So instead compare this:
![image](https://github.com/googlefonts/fontmake-rs/assets/6466432/16a818a3-bf26-4a20-82da-19f8b2caaa2d)

Reply:
Added the check to not adjust names <= 255. Closing, as always please reopen if you disagree.